dev-utils/src/commands/init_command.rs

```diff
@@ -0,0 +1,225 @@
+use super::genesis_command::GenesisCommand;
+
+use failure::Error;
+
+use super::genesis_dev_config::GenesisDevConfig;
+use read_input::prelude::*;
+use serde_json::json;
+use serde_json::Value as JsonValue;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct InitCommand {
+    data: String,
+    genesis_time: JsonValue,
+    app_hash: JsonValue,
+    app_state: JsonValue,
+    genesis_dev: JsonValue,
+    tendermint_pubkey: String,
+    staking_account_address: String,
+    distribution_addresses: Vec<String>,
+}
+
+impl InitCommand {
+    pub fn new() -> Self {
+        InitCommand {
+            genesis_time: json!(null),
+            app_hash: json!(null),
+            app_state: json!(null),
+            genesis_dev: json!(null),
+            tendermint_pubkey: "".to_string(),
+            staking_account_address: "".to_string(),
+            distribution_addresses: vec![],
+
+            data: serde_json::to_string(&GenesisDevConfig::new()).unwrap(),
+        }
+    }
+
+    pub fn read_wallet(&mut self, id: &str, default1: &str, default2: &str) {
+        let obj = self.genesis_dev.as_object_mut().unwrap();
+        let distribution = obj["distribution"].as_object_mut().unwrap();
+        let a = input()
+            .msg(format!("wallet {}({}) address=", id, default1))
+            .default(default1.to_string())
+            .get();
+        let b = input()
+            .msg(format!("wallet {}({}) amount=", id, default2))
+            .default(default2.to_string())
+            .get();
+        distribution.insert(a.to_string(), json!(b));
+        self.distribution_addresses.push(a.to_string());
+    }
+    pub fn read_information(&mut self) {
+        self.genesis_dev =
+            serde_json::from_str(&self.data).expect("failed to parse genesis dev config");
+
+        self.staking_account_address = input()
+            .msg("Please staking_account_address(0x3ae55c16800dc4bd0e3397a9d7806fb1f11639de)=")
+            .default("0x3ae55c16800dc4bd0e3397a9d7806fb1f11639de".to_string())
+            .get();
+        self.read_wallet(
```

Comment:
I've just tried running it locally and this part is a bit unclear to me -- what is "wallet 1", "wallet 2", "wallet 3", ...?

For the local development, I think one just needs one wallet / faucet (and perhaps the wallet of the validator)... From what I'm guessing, these "wallets" refer to some of the necessary addresses in the config (e.g. of the validator, initial rewards pool) -- I think for the local development, they are a bit irrelevant (e.g. all of the amount in "wallet 1" goes to the rewards pool anyway, so even if I had a private key, it'll be useless).

Reply:
because total amount is fixed, i followed the same wallet convention.(5 wallets)

Reply:
but the Quickstart tool can calculate how much to put in

Reply:
The amounts can be whatever as long as the distribution adds up to the total supply

Reply:
ok, i will change total suppy.

Reply:
adding wallet proceed until total money is distributed.
so user can make any number of wallets.(but should be over 3 )